website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-enterprise-sso-with-azure-active-directory.md

```diff
@@ -157,3 +169,11 @@ Users in your Azure AD account will now be able to log into the application
 by navigating to the URL:
 
 `https://cloud.getdbt.com/enterprise-login/<login-slug>`
+
+### Troubleshooting Tips
+
+Ensure that the domain name under which user accounts exist in Azure matches the domain supplied in the SSO configuration on the dbt side.
+
+<Lightbox collapsed="true" src="/img/docs/dbt-cloud/dbt-cloud-enterprise/azure/azure-get-domain.png" title="Obtaining the user domain from Azure" />
```

Comment:
Did you want to include the Troubleshooting Tips in the same box? 
![image](https://user-images.githubusercontent.com/46451573/103547733-cbf4fa00-4e72-11eb-8668-55bd299f9115.png)